packages/@aws-cdk/aws-redshift/lib/cluster.ts

```diff
@@ -662,4 +675,31 @@ export class Cluster extends ClusterBase {
       throw new Error('Cannot add a parameter to an imported parameter group.');
     }
   }
+
+  /**
+   * Adds default IAM role to cluster
+   *
+   * @param defaultIamRole the IAM role to be set as the default role
+   */
+  public addDefaultIamRole(defaultIamRole: iam.IRole): void {
+    const defaultRoleCustomeResource = new AwsCustomResource(this, 'default-role', {
```

Comment:
why is there an e in custom here? Can we change this to be just "Custom"?

Reply:
Fixed spelling mistake